from_base10/main.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+func fromBase10(num int, toBase int) string {
+	var baseB string
+	if 2 <= toBase && toBase <= 36 {
+		k, power := 1, -1 // because "for" makes 1 extra move
+		for ; k <= num; k = k * toBase {
+			power++
+		}
+
+		l := int(math.Pow(float64(toBase), float64(power)))
```

Comment:
You are repeating this expression multiple times in the function. I'd declare it as a function.

Reply:
I did it.